src/core/api_client/tests/factories/mock.factory.ts

```
@@ -231,43 +237,22 @@ export class MockFactory {
-      AuthFactory.getApiUrl(path),
-      (url: string, opts: any) => {
-        this.callCount++;
-        return {
-          throws: new ApiClientError("Can't connect to server", {
-            kind: 'cannot-connect',
-            temporary: true,
-          }),
-        };
-      },
-    );
+  static setupNetworkError(path: string, method: string = 'GET'): void {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)